website/js/average-words.js

```diff
@@ -1,9 +1,12 @@
 function addVisWord(media, avgWords) {
-  var selector = ".avgWords-container ." + media+'-container';
-  var header = '<h3>' + mediaEN2C[media] + '<\/h3><span>' + avgWords +
+  var media_Chinese = mediaEN2C[media];
```

Comment:
Also, should not use the global variable. The `mediaEN2C` should be passed by the arguments.